examples/stable_diffusion_v2/lora/lora.py

```diff
@@ -0,0 +1,234 @@
+'''
+inherited from mindpet 
+https://github.com/mindspore-lab/mindpet/blob/master/tk/delta/lora.py
+
+'''
+
+
+import math
+# from tk.utils.version_utils import is_version_ge
+
+import mindspore as ms
+import mindspore.nn as nn
+from mindspore import ops
+from mindspore import Parameter
+from mindspore.common.tensor import Tensor
+from mindspore import dtype as mstype
+from mindspore.ops import operations as P
+from mindspore.ops import functional as F
+from mindspore.common.initializer import initializer, HeUniform
+# from tk.delta.delta_constants import VALID_TENSOR_DATATYPE
+from mindspore import dtype as mstype
+VALID_TENSOR_DATATYPE = [mstype.float16, mstype.float32]
+
+def is_version_ge(current_version, base_version):
```

Comment:
Please remove this function, referring to the last comment for version check.

Reply:
ok